### ISSUE TITLE
Traductions fr manquantes

### ISSUE THREAD

**Issue by Xanatos00** [MT-Eurythmia/Eurythmia-Evolution#24]
Pour les traductions de mods manquantes, serait-il possible d'avoir, en attendant MAJ , un fichier patch qui centraliserait les chaînes manquantes traduites sur l'ensemble du jeu ?
Fichier sur lequel nous pourrions pull request.

**Comment by paly2** [comment]
Oui, bien sûr, toute PR serait bienvenue ! Je suggère de rajouter les traductions manquantes dans le dossier `locale` du mod `misc`.

**Comment by Xanatos00** [comment]
cf. [réponse PR](https://github.com/MT-Eurythmia/Eurythmia-Evolution/pull/26#issuecomment-757451415)

Je comprends mieux maintenant les traductions manquantes correspondent à première vue au non support de `intllib`
 > mods/3d_armor/3d_armor/intllib.lua
mods/basic_signs/intllib.lua
mods/bonemeal/intllib.lua
mods/castle_gates/intllib.lua
mods/farming/intllib.lua
mods/mob_horse/intllib.lua
mods/mobs_animal/intllib.lua
mods/mobs_monster/intllib.lua
mods/mobs_npc/intllib.lua
mods/mobs_redo/intllib.lua
mods/moreblocks/intllib.lua
mods/pvpplus/intllib.lua
mods/signs_lib/intllib.lua
mods/travelnet/intllib.lua
